index.bs

```diff
@@ -382,6 +382,13 @@ When this method is invoked, the user agent MUST run the following steps:
               <dd> Append |session| to the [=list of inline sessions=].
             </dl>
         1. [=/Resolve=] |promise| with |session|.
+        1. [=Queue a task=] to perform the following steps:
```

Comment:
Does the `Resolve` step wait until the task is done?

Reply:
It's the other way around, `resolve` uses a microtask, and `queue a task` uses a regular task, so we can guarantee that the callback attached in `requestSession(..).resolve(callback)` will execute before the task below does.

Reply:
ok. Resolve actually calls the `then` function. I think this looks good!

Reply:
Yep!

er, i should have written `then` instead of `resolve` in my comment above, but you get the idea